lib/util/storage_manager.dart

```diff
@@ -5,6 +5,8 @@ class StorageManager {
     final SharedPreferences prefs = await SharedPreferences.getInstance();
     if (value is int) {
       await prefs.setInt(key, value);
+    } else if (value is double) {
```

Comment:
Do we need that anywhere or did the exclusion just bug you?

Reply:
Yes that was just annoying lol - we listed all the data types that `SharedPreferences` supports, __except__ double.